sharkfuser/include/fusili/graph.h

```diff
@@ -143,6 +149,39 @@ Graph::convFProp(const std::shared_ptr<TensorAttr> &x,
   return y;
 }
 
+inline ErrorOr<std::string> Graph::emitVMFB() {
+  FUSILI_LOG_LABEL_ENDL("INFO: Emitting VMFB for graph");
+
+  // Write input asm to temp file.
+  TempFile input = TRY(TempFile::create("iree_compile_input"));
+  FUSILI_CHECK_ERROR(input.write(this->emitAsm()));
```

Comment:
Should `emitAsm()` return an `ErrorOr<T>` type when the graph fails to validate?